pilot/pkg/bootstrap/server.go

```diff
@@ -608,7 +608,7 @@ func (s *Server) initSecureGrpcServer(port string, keepalive *istiokeepalive.Opt
 
 	cfg := &tls.Config{
 		Certificates: []tls.Certificate{certP},
-		ClientAuth:   tls.VerifyClientCertIfGiven,
+		ClientAuth:   tls.NoClientCert,
```

Comment:
@costinm @myidpt @howardjohn Can you please address your thoughts on this? The original PR has unresolved issues concerning this. On top of that, the original PR was never merged, so this isn't really a "cherrypick".